io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -71,17 +76,18 @@ private:
    TBufferMerger(const TBufferMerger &);
 
    /** TBufferMerger has no copy operator */
-   TBufferMerger& operator=(const TBufferMerger&);
+   TBufferMerger &operator=(const TBufferMerger &);
 
    void Push(TBufferFile *buffer);
-   void Listen();
+   void WriteOutputFile();
 
-   std::mutex fFilesMutex;                                       //< Mutex used to lock fAttachedFiles
+   const std::string fName;
+   const std::string fOption;
+   const Int_t fCompress;
```

Comment:
Yes, what @pcanal said, and it makes it easy to catch bugs if these members are ever modified by mistake.